themes/bootstrap3/templates/Recommend/SummonResults.phtml

```diff
@@ -1,7 +1,7 @@
 <?php $searchObject = $this->recommend->getResults(); ?>
 <?php $results = $searchObject->getResults(); ?>
 <?php if (!empty($results)): ?>
-  <h4><?=$this->transEsc('Summon Results')?></h4>
+  <h2><?=$this->transEsc('Summon Results')?></h2>
```

Comment:
I can't test Summon

Reply:
I'm not sure if we have any Summon users left, but I'd be willing to trust this change. ;-)